packages/canvas/render/src/page-block-function/accessor-map.ts

```
@@ -0,0 +1,54 @@
+import { generateFunction } from '../data-utils'
+import { globalNotify } from '../canvas-function'
+
+type IAccessorType = 'getter' | 'setter'
+interface IAccessor {
+  getter: { value: string }
+  setter: { value: string }
+}
+
+export function useAccessorMap(context) {
+  const generateAccessor = (type: IAccessorType, accessor: IAccessor, property: string) => {
+    const accessorFn = generateFunction(accessor[type].value, context) as Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)